src/css/custom.css

```
@@ -633,3 +633,323 @@ table {
   margin: 0px;
   font-weight: 600;
 }
+
+.section-container {
+  position: relative;
+  overflow: hidden;
+  padding: 75px 0px;
+}
+
+@media (min-width: 768px) {
+  .section-container {
+    padding: 100px 0px;
+  }
+}
+
+.section-container__alternative-background {
+  background-color: var(--theme-color-grey) !important;
+}
+
+.pricing-card__title {
+  font-weight: 900;
+  font-size: 28px;
+  margin: 10px 0px;
+}
+
+.pricing-card__price {
+  font-size: 28px;
+  margin: 10px 0px;
+}
+
+.pricing-card__talk-icon {
+  width: 45px;
+  height: 45px;
+}
+
+.pricing-card__talk-icon svg {
+  width: 100%;
+  height: 100%;
+}
+
+.pricing-card__denominator {
+  font-size: 14px;
+  font-weight: 100;
+  vertical-align: super;
+  letter-spacing: 0.5px;
+  margin-left: 2px;
+}
+
+.pricing-card__bulletpoint-container {
+  display: flex;
+  align-items: center;
+  gap: 10px;
+}
+
+.pricing-card__link {
+  display: flex;
+  justify-content: center;
+}
+
+.pricing-card__link:hover {
+  text-decoration: none !important;
+}
+
+.pricing-card__card {
+  position: relative;
+  display: flex;
+  flex-direction: column;
+  border-radius: 36px;
+  width: 100%;
+  padding: 20px;
+  transition: all 0.2s ease-out;
+}
+
+.pricing-card__card:hover {
+  transform: scale(1.03);
+}
+
+.pricing-card__badge {
+  position: absolute;
+  left: 50%;
+  top: 0px;
+  transform: translate(-50%, -50%);
+  font-size: 14px;
+  padding: 8px 32px;
+  border-radius: 32px;
+  font-weight: 500;
+  white-space: nowrap;
+  color: white;
+}
+
+.pricing-card__benefits {
+  display: flex;
+  flex-flow: column;
+  flex-grow: 1;
+  gap: 10px;
+  justify-content: center;
+  padding: 15px 0px;
+  margin-bottom: 15px;
+}
+
+.pricing-card__bulletpoint {
+  border-radius: 32px;
+  width: 23px;
+  height: 23px;
+  flex-shrink: 0;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.pricing-card__bulletpoint svg {
+  width: 75%;
+  height: 75%;
+}
+
+.pricing-card__benefit-button {
+  transition: font-weight 0.2s ease-in-out;
+  background: none;
+  color: inherit;
+  border: none;
+  padding: 0;
+  font: inherit;
+  cursor: pointer;
+  outline: inherit;
+}
+
+.pricing-card__benefit-button:hover {
+  font-weight: 600;
+}
+
+.list-card__card {
+  width: 100%;
+  padding: 36px 48px;
+  background-color: #eee;
+  border-radius: 8px;
+}
+
+@media (max-width: 768px) {
+  .list-card__card {
+    padding: 24px;
+  }
+}
```

Can we rearrange it and unify so that each media query is defined as from `min-width`?
```suggestion
.list-card__card {
  width: 100%;
  padding: 24px;
  background-color: #eee;
  border-radius: 8px;
}

@media (min-width: 768px) {
  .list-card__card {
    padding: 36px 48px;
  }
}
```

---

Sure!